README.md

```diff
@@ -1,252 +1,34 @@
 # Liquibase Linter
-Liquibase extension to add configurable script linting.
+
+*Quality control for your Liquibase scripts*
 
 [![Build Status](https://travis-ci.org/whiteclarkegroup/liquibase-linter.svg?branch=master)](https://travis-ci.org/whiteclarkegroup/liquibase-linter) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/320a8a4be4fd44feb9d6102ccdc7e240)](https://www.codacy.com/project/whiteclarkegroup/liquibase-linter/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=whiteclarkegroup/liquibase-linter&amp;utm_campaign=Badge_Grade_Dashboard)
 
-## Setup
-To add `liquibase-linter` into your project just add the artifact as a dependency to the `liquibase-maven-plugin`
-The extension will automatically be picked up by liquibase and the rules applied.
+## Get Started
+
+**1** Add `liquibase-linter` to your pom as a dependency of `liquibase-maven-plugin`:
 
 ```xml
 <plugin>
     <groupId>org.liquibase</groupId>
     <artifactId>liquibase-maven-plugin</artifactId>
     <configuration>
-        <propertyFile>${liquibase.property.file}</propertyFile>
+        ...
     </configuration>
     <dependencies>
         <dependency>
-            <groupId>com.whiteclarkegroup</groupId>
+            <groupId>com.whiteclarkegroup.liquibase</groupId>
```

Comment:
This should be com.whiteclarkegroup only

Reply:
Nice catch, fixed!